src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -15,11 +15,31 @@
         "If this problem persists, please contact us."
     );
 
-    public static final List<String> EXPORTED_TO_V2 = Arrays.asList(
-        "This Overleaf project has been moved to Overleaf v2, and git access is temporarily unsupported.",
-        "",
-        "See https://www.overleaf.com/help/342 for more information."
-    );
+    static List<String> buildExportedToV2Message(String remoteUrl) {
+        if (remoteUrl == null) {
+            return Arrays.asList(
+                    "This Overleaf project has been moved to Overleaf v2, and cannot be used with git at this time.",
```

Comment:
Grammar nitpick: we should not have a comma before `and` here, because the latter clause is not an independent clause. (Alternatively, it could read "it cannot".)

Reply:
Good catch :+1: